packages/components/atoms/f-form-field/src/components/_tests/FormField.test.js

```diff
@@ -254,6 +283,24 @@ describe('FormField', () => {
                 expect(formInput.attributes('class')).toContain('c-formField-field--noFocus');
             });
         });
+
+        describe('isDisabled :: ', () => {
+            it.each([
+                [true, 'disabled'],
+                [false, null]
+            ])('should return %s if disabled attribute is %s', (expected, disabled) => {
+                // Arrange
+                const propsData = {
+                    disabled
+                };
+
+                // Act
+                const wrapper = shallowMount(FormField, { propsData });
+
+                // Assert
+                expect(wrapper.vm.isDisabled).toEqual(expected);
```

Comment:
I am not a huge fan of testing internal implementation details, this leads to brittle tests. In fact its recommended not to do this in the vue test utils guide. https://vue-test-utils.vuejs.org/guides/#getting-started

Reply:
We should only be testing the public interface in my opinion.

Reply:
Apart from this it looks good, I will tick, but think we should double tink about testing things like this

Reply:
@matthewhardern I'm doing a few more tickets on form field, I'll go over the tests within the next tickets?